fix_includes.py

```diff
@@ -2169,6 +2172,19 @@ def main(argv):
   parser.add_option('--nosafe_headers', action='store_false',
                     dest='safe_headers')
 
+  # --reorder and --no-reorder share the same destination variable.  Omitting
+  # the default here would still produce the intended behavior, however if
+  # the code is re-organized such that these two specifications are swapped
+  # relative to each-other there will be a change in behavior.
+  parser.add_option('--reorder', default=False, action='store_true', dest='reorder',
```

Comment:
Sorry to complain again but the 2nd part is wrong: Omitting the default here (and only here) would not change anything even when reordering stuff. Both set the default to `False`: This one implicitly (if default removed) and the other one explicitly. Just remove the part starting with `Omitting...` would be fine with me, keep the default if you like, but again: It's completely optional here.

Reply:
No worries, I'm not easily offended.  I am, however, a little slow sometimes; I understand now what you were getting at.